packages/shared/src/Logger.js

```diff
@@ -72,18 +74,24 @@ const HotelGalleryType = {
 };
 
 type LogHotelGalleryType = $Keys<typeof HotelGalleryType>;
+type HotelsBookNowArgs = {|
+  +hotelID: string,
+  +rooms: number,
+  +guests: number,
+  +price: number,
+|};
 
 function hotelsGalleryOpened(type: LogHotelGalleryType) {
   LoggingModule.hotelsGalleryOpened(type);
 }
 
-function hotelsBookNowPressed(
-  hotelID: string,
-  rooms: number,
-  guests: number,
-  price: number,
-  formattedPrice: string,
-) {
+async function hotelsBookNowPressed({
+  hotelID,
+  rooms,
+  guests,
+  price,
+}: HotelsBookNowArgs) {
+  const formattedPrice = await currencyFormatter(price, 'EUR');
```

Comment:
Is it always going to be `EUR`?

Reply:
Yes, I guess that makes most sense, or we could change it to CZK later if they prefer that.

Reply:
But we are getting the currency there as well: https://github.com/kiwicom/mobile/pull/1415/files#diff-6b6ced58fb2208dc5f52e08d6530f058R38, so I am confused why this value needs to be hardcoded?

Reply:
we always get the price in EUR from booking.com. The currencyFormatter takes price in euro and formats it to the desired currency. 

```
const price = 150 // This is price in euro
const currency = 'EUR'
const formattedCurrency = currencyFormatter(price, currency);
console.log(formattedCurrency)
// 150€
```

In this case, this is just what we want to log. This formatting is for us. I don't see why we would log the price in the users currency.

Reply:
hmm, now I got confused. let me double check that

Reply:
Sorry, I was wrong. 

I will change this.